.storybook/test-runner.js

```diff
@@ -0,0 +1,43 @@
+const {getStoryContext} = require('@storybook/test-runner');
+const {injectAxe, configureAxe, checkA11y} = require('axe-playwright');
+
+const a11yConfig = {
+  async preRender(page, context) {
+    await injectAxe(page);
+  },
+  async postRender(page, context) {
+    // Get entire context of a story, including parameters, args, argTypes, etc.
+    const storyContext = await getStoryContext(page, context);
+
+    // Allow individual stoies to opt out of axe-playwright tests by setting story parameters.
+    if (storyContext.parameters?.a11y?.disable) {
+      return;
+    }
+
+    await configureAxe(page, {
+      rules: [
+        {
+          id: 'landmark-one-main',
+          enabled: false,
+        },
+        {
+          id: 'page-has-heading-one',
+          enabled: false,
+        },
+        {
+          id: 'region',
+          enabled: false,
+        },
+        {
+          id: 'scrollable-region-focusable',
+          enabled: false,
+        },
```

Comment:
suggestion: can we get comments above each of these for why we decided to disable them please?